toolkit/tools/scheduler/schedulerutils/buildworker.go

```diff
@@ -193,11 +197,44 @@ func buildSRPMFile(agent buildagents.BuildAgent, buildAttempts int, srpmFile, ou
 	)
 
 	logBaseName := filepath.Base(srpmFile) + ".log"
-	err = retry.Run(func() (buildErr error) {
-		builtFiles, logFile, buildErr = agent.BuildPackage(srpmFile, logBaseName, outArch, dependencies)
-		return
-	}, buildAttempts, retryDuration)
-
+	logger.Log.Debugf("osamatest: with check is (%t)", agent.Config().RunCheck)
+	maxAttempts := 3
```

Comment:
Should I keep this here or make the change from time.sleep to waitgroups in retry.run so that it applies project-wide?